action/messageHubProduce.py

```diff
@@ -83,23 +83,20 @@ def main(params):
     # we successfully connected and found the topic metadata... let's send!
     if producer is not None:
         try:
-            logging.info("Producing message")
-
             # only use the key parameter if it is present
             value = validatedParams['value']
+            batch = isinstance(value, list)
```

Comment:
Could maybe require a values parameter instead of value for batched messages.

Reply:
Would need to error if arguments are passed to both parameters in that case though.